frontend/packages/api/tests/clients/admin.test.ts

```diff
@@ -6,7 +6,7 @@ interface Context {
 	client: ReturnType<typeof createAdminClient>
 }
 
-describe<Context>('Admin Client', (adminClient) => {
+describe.skip<Context>('Admin Client', (adminClient) => {
```

Comment:
why do you skip this test

Reply:
reworked the whole thing, tests are fixed.